perception_dataset/deepen/download_paint_annotations_to_pcd.py

```diff
@@ -0,0 +1,95 @@
+import argparse
+from datetime import date
+import json
+import os
+from typing import List
+import zlib
+
+import numpy as np
+import requests
+from sensor_msgs.msg import PointField
+import tqdm
+import yaml
+
+CLIENT_ID = os.environ["DEEPEN_CLIENT_ID"]
+ACCESS_TOKEN = os.environ["DEEPEN_ACCESS_TOKEN"]
+today = str(date.today()).replace("-", "")
+
+NUM_DIMENSIONS = 5
+
+
+def get_datasets(dataset_id: str, input_base_dir: str):
+    headers = {
+        "Authorization": f"Bearer {ACCESS_TOKEN}",
+        "Content-Type": "application/json",
+    }
+
+    if dataset_id is None:
+        print("Dataset ID not found")
+        return
+
+    DATASET_URL = f"https://tools.deepen.ai/api/v2/datasets/{dataset_id}/label_types/3d_point/paint_labels?stageId=QA"
+
+    response = requests.get(DATASET_URL, headers=headers)
```

Comment:
It looks better to end function by return if response contains HTTP status error.